### DISCUSSION TITLE
quarto CLI install fails on MacOS Ventura, M1 chip

### DISCUSSION THREAD

**Discussion by AnnaGolubeva** [quarto-dev/quarto-cli#4037] [Q&A]
Quarto CLI installation fails on my macbook pro with M1 chip with Ventura 13.1. I am not sure whether the issue is M1 or Ventura OS. Can someone confirm? Is there a solution?

**Comment by dragonstyle** [comment]
Hi, thanks for letting us know. Can you give us any more details about the error / failure - we aren't currently aware of issues installing in that configuration.

**Reply by AnnaGolubeva** [reply]
Hi!
I download [quarto-1.2.313-macos.pkg](https://github.com/quarto-dev/quarto-cli/releases/download/v1.2.313/quarto-1.2.313-macos.pkg), install and get this error message:
<img width="620" alt="Screenshot 2023-01-20 at 11 55 14 AM" src="https://user-images.githubusercontent.com/27358382/213793645-b89b077c-d5cf-4b84-9714-250e90875e57.png">

Let me know what other info could be useful.

**Reply by cscheid** [reply]
Ouch, that's not a very helpful message, sorry. Unfortunately I can't seem to reproduce this (I'm on Ventura and M1):

<img width="609" alt="image" src="https://user-images.githubusercontent.com/285675/213794930-2cd17e47-3189-4d37-9bc6-26d3bfd0b34f.png">

Is it possible (for example) that you had an older installation from a different user, and now there's a new user without permissions to overwrite the old files?

**Reply by AnnaGolubeva** [reply]
Hmm, I was almost sure Ventura is the culprit!
I am the only user of this macbook and I'm rather certain that I did not install Quarto previously (unless it sneaked in as a dependency with another software, if that's possible at all!).

I'm wondering, what else could it conflict with?

**Reply by cscheid** [reply]
(just to be clear, I'm grasping at straws here, so I'm sorry if these questions are obvious or dumb. This is the first time we've ever seen this error!)

- Do you have enough disk space?
- Did you open the installer from a shell running at lowered privileges?
- Do you have security software that might be preventing us from writing to `/Applications/quarto`? (for example, is your laptop managed by your company?)

**Reply by AnnaGolubeva** [reply]
No worries, all questions are welcome, since I'm clueless and don't know how to debug this issue at all. Thank you for trying to help, I appreciate it!

- yes, a lot of free disk space (870.88 GB available)
- I did not use the shell for installation, just opened it by clicking (on this note, I was wondering, is it possible to install quarto in a venv?)
- no security software, it's my personal laptop

Let me give you a little more context that might or might not be helpful: I intended to use quarto with VS code. I fist installed the extension in VS code, then tried creating a project and realized that I need to install the Quarto CLI (I think a message popped up and sent me to the Quarto website where I downloaded the said package, which then failed to install).

**Reply by dragonstyle** [reply]
The package installer should write to a log file at:

`/var/log/install.log`

This is a global install log so will contain installation logs for Quarto and other programs as well (in case you'd like to filter it or purge it to create a Quarto only version of the log file). Would you mind either having a look in that file for additional information or sharing it here (or with me directly via email charles at posit dot co)?

**Reply by AnnaGolubeva** [reply]
Nice, that helps! I suspect the following might be the issue:
> Architecture Translation: Distribution failed architecture check and is about to be re-executed as Intel.
Though I don't know how to solve it.

I will send you the file with complete log around quarto in a moment.

**Reply by LukeGahan** [reply]
Hi, @AnnaGolubeva. I ran into the same issue but managed to work around it by using a terminal running via rosetta 2 to force mac to install as an application running on rosetta 2. Steps for the workaround are:

1. Set up a terminal to run on rosetta 2 if you haven't done this before: [how-to here](https://osxdaily.com/2020/11/18/how-run-homebrew-x86-terminal-apple-silicon-mac/ )
2. Using your rosetta terminal, install using the following command `sudo installer -pkg quarto-1.2.313-macos.pkg -target ~/Applications/` (adjusting pkg name and install location as needed). 


That should get you going until the issue is resolved.

**Reply by AnnaGolubeva** [reply]
Thanks for your answer, @LukeGahan! In my case it turned out to be even easier: I had to simply move the installation package from Downloads to the Desktop - a fix suggested by @dragonstyle (Thanks again!)
